### PR TITLE
PLU-257: chore: change login page layout

### PR DESCRIPTION
**Pull request by m0nggh**
## Details
- Changed login page layout
- Logo becomes an image instead of a link back to homepage...

TODO:
- [x] Get new graphic from Stacey
- [x] Test e2e on staging

## Before & After Screenshots
**Before**:

https://github.com/opengovsg/plumber/assets/65110268/3e89ab53-2e1e-45a4-b908-84eb626822e3

**After**:

https://github.com/opengovsg/plumber/assets/65110268/c971fd8f-8b65-40e6-a83a-c75d93d1e388

## Tests
- [x] Can still login using OTP or singpass/sgid

### CODE REVIEW

File: packages/frontend/src/pages/Login/index.tsx [hunk 1]
```diff
@@ -1,99 +1,49 @@
...
-              <FormattedMessage id="brandText" />
-            </Typography>
-          </Box>
-          <LoginForm />
-        </Container>
-      </Box>
-    </Box>
+        <Flex flexDir="column" gap={2} pl={11}>
+          <Text textStyle="heavy" maxW="60%">
+            Automate your manual and repititive tasks
```
Comment (liangyuanruo) [inline packages/frontend/src/pages/Login/index.tsx]: typo - maybe install a spellchecker in your IDE?
```suggestion
            Automate your manual and repetitive tasks
```

**Review by liangyuanruo — APPROVED** [review]
approved with comment

File: packages/frontend/src/pages/Login/index.tsx (hunk 1 above)
Comment (m0nggh) [reply packages/frontend/src/pages/Login/index.tsx]: ok will do, thanks!

File: packages/frontend/src/pages/Login/index.tsx [hunk 2]
```diff
@@ -1,99 +1,49 @@
...
+            A no-code tool that connects multiple applications together to
+            automate your workflows.
+          </Text>
+        </Flex>
+
+        <Show above="md">
+          <Center pr={8}>
+            <Image
+              src={landingImg}
+              w={{ base: '80%', lg: '60%', '2xl': '65%' }}
```
Comment (pregnantboy) [inline packages/frontend/src/pages/Login/index.tsx:34]: can we set a maxW of 26rem here

**Review by pregnantboy — APPROVED** [review]
lgtm, just set a maxW to the login page img

File: packages/frontend/src/pages/Login/index.tsx (hunk 2 above)
Comment (m0nggh) [reply packages/frontend/src/pages/Login/index.tsx:34]: yes boss looks better

**Pull request merged**